litmus-portal/backend/graphql-server/pkg/database/mongodb/init.go

```diff
@@ -6,17 +6,18 @@ import (
 	"os"
 	"time"
 
-	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
 var clusterCollection *mongo.Collection
+var workflowRunCollection *mongo.Collection
 var backgroundContext = context.Background()
 var err error
 
 var collection = map[string]string{
```

Comment:
> 
```suggestion
var collections = map[string]string{
```
I forgot to make it plural